microservices/first/utils/db_manager.py

```diff
@@ -0,0 +1,127 @@
+import json
+import sqlite3
+from sqlite3 import Error
+from typing import Any, List, Optional
+
+class DBManager:
```

Comment:
Can you add some documentation somewhere specifying what the table schemas look like, just for future reference? You can add it as a class specification, a module specification, or wherever else you think is best